### PR TITLE
Added Helm chart

### PR DESCRIPTION
**Pull request by fheinecke**
This PR adds a basic Helm chart for the project with Prometheus operator and Grafana support. I'll add deployment logic in a follow-up PR if this change looks good to reviewers. I've done some basic testing to ensure that the chart can at least render properly with options enabled, but I haven't tested much more than that.

Note: this PR is dependent on https://github.com/gravitational/aws-quota-checker/pull/28 merging first.

### CODE REVIEW

**Review by r0mant — APPROVED** [review]

**Review by taraspos — APPROVED** [review]

**Comment by fheinecke** [comment]
I'm probably going to leave this open for a couple more days if you folks don't mind. I'd like to manually deploy this to one of the clusters I manage just to double check that it works properly.

**Comment by fheinecke** [comment]
This should be good to merge once these PRs land:
* https://github.com/gravitational/aws-quota-checker/pull/33
* https://github.com/gravitational/aws-quota-checker/pull/34
* https://github.com/gravitational/aws-quota-checker/pull/35

This now also adds configurable alerts for both long-running AWS queries, and resources nearing their limit.

I have this deployed to the dev GHA EKS cluster (accessible via https://platform.teleport.sh). If desired, these changes can be previewed there.

**Pull request merged**
